integration-tests/modules/__tests__/shipping-options/store/shipping-options.spec.ts

```diff
@@ -165,35 +190,77 @@ medusaIntegrationTestRunner({
                   region_id: region.id,
                   amount: 1100,
                 },
+                {
+                  region_id: regionTwo.id,
+                  amount: 500,
+                },
               ],
               rules: [],
             },
             adminHeaders
           )
         ).data.shipping_option
+      })
 
-        cart = (
-          await api.post(
-            `/store/carts`,
-            {
-              region_id: region.id,
-              sales_channel_id: salesChannel.id,
-              currency_code: "usd",
-              email: "test@admin.com",
-              items: [
-                {
-                  variant_id: product.variants[0].id,
-                  quantity: 1,
-                },
-              ],
-            },
+      describe("GET /admin/shipping-options?cart_id=", () => {
+        it("should get shipping options for a cart successfully", async () => {
+          cart = (
+            await api.post(
+              `/store/carts`,
+              {
+                region_id: region.id,
+                sales_channel_id: salesChannel.id,
+                currency_code: "usd",
+                email: "test@admin.com",
+                items: [
+                  {
+                    variant_id: product.variants[0].id,
+                    quantity: 1,
+                  },
+                ],
+              },
+              storeHeaders
+            )
+          ).data.cart
+
+          const resp = await api.get(
+            `/store/shipping-options?cart_id=${cart.id}`,
             storeHeaders
           )
-        ).data.cart
-      })
 
-      describe("GET /admin/shipping-options?cart_id=", () => {
-        it("should get all shipping options for a cart successfully", async () => {
+          const shippingOptions = resp.data.shipping_options
+
+          expect(shippingOptions).toHaveLength(1)
+          expect(shippingOptions[0]).toEqual(
+            expect.objectContaining({
+              id: shippingOption.id,
+              name: "Test shipping option",
+              amount: 1100,
+              price_type: "flat",
+            })
+          )
+        })
+
+        it("should get shipping options with region prices  for a cart successfully", async () => {
```

Comment:
Q: Both tests are doing the same thing, Maybe we could just merge them and do something like
- create cart with a region
- retrieve shipping options
- expectations
- update cart region
- retrieve shipping options
- expectation

I find it confusing to have two different tests that does the same thing except that if you read the description of the first one we don't know that we are looking for region prices as well. 

Let me know if I overlooked at something 👍

Reply:
Sure, we can merge the two